x/distribution/simulation/proposals.go

```diff
@@ -0,0 +1,51 @@
+package simulation
+
+import (
+	"math/rand"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/distribution/keeper"
+	"github.com/cosmos/cosmos-sdk/x/distribution/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+)
+
+// OpWeightSubmitCommunitySpendProposal app params key for community spend proposal
+const OpWeightSubmitCommunitySpendProposal = "op_weight_submit_community_spend_proposal"
+
+// ProposalContents defines the module weighted proposals' contents
+func ProposalContents(k keeper.Keeper) []simulation.WeightedProposalContent {
+	return []simulation.WeightedProposalContent{
+		{
+			AppParamsKey:       OpWeightSubmitCommunitySpendProposal,
+			DefaultWeight:      5,
```

Comment:
I think the Default weights for proposals should also be in `simapp/params/weights.go` or parsed from simulation param file; just like with default weights on msgs